docs/guides/auth.md

```diff
@@ -553,3 +553,50 @@ $opts = [
 ];
 $client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
 ```
+
+### Dart
+
+#### Base case - no encryption or authentication
+
+```dart
+final channel = new ClientChannel('localhost', port: 50051,
+    options: new ChannelOptions.insecure());
+final client = new GreeterClient(channel);
+```
+
+#### With server authentication SSL/TLS
+
+```dart
+// Load a custom roots file.
+final trustedRoot = new File('roots.pem').readAsBytesSync();
+final channelCredentials =
+    new ChannelOptions.secure(certificate: trustedRoot);
```

Comment:
Seems to have changed to `certificates`.

https://github.com/grpc/grpc-dart/blob/master/lib/src/client/options.dart#L71

Reply:
Fixed